src/mono/mono/mini/intrinsics.c

```diff
@@ -1058,7 +1058,7 @@ mini_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 			EMIT_NEW_TEMPLOAD (cfg, ins, span->inst_c0);
 			return ins;
 		} else
-			return NULL;
+			return emit_runtime_helpers_intrinsics (cfg, cmethod, fsig, args);
```

Comment:
Half of the runtime helpers is handled inline here by existing code and the other half is in `emit_runtime_helpers_intrinsics` now. Move everything to `emit_runtime_helpers_intrinsics` method?

Reply:
Yeah I've planned to unify that but:
a) I wanted to ask the mono maintainer whether i should move everything into that method or out of it
b) I've also wanted to get everything working first with minimal changes